lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -573,8 +573,9 @@ class DifferentiationTask {
   /// `NestedApplyInfo`s.
   DenseMap<ApplyInst *, NestedApplyInfo> nestedApplyInfo;
 
-  /// Mapping from original `struct_extract` instructions to their strategies.
-  DenseMap<StructExtractInst *, StructExtractDifferentiationStrategy>
+  /// Mapping from original `struct_extract` and `struct_element_addr`
+  /// instructions to their strategies.
+  DenseMap<SILInstruction *, StructExtractDifferentiationStrategy>
```

Comment:
I didn't change the names of `StructExtractDifferentiationStrategy` or `getStructExtractDifferentiationStrategies` because I couldn't think of anything more suitable.